src/components/Select/components/SelectMenu/SelectMenu.style.tsx

```diff
@@ -32,8 +32,9 @@ export const menuStyle = ({ status, size }: Props) => (theme: Theme): Serialized
   top: ${status !== 'normal' ? '70%' : '110%'};
   z-index: 500;
   position: absolute;
-  min-width: 220px;
   max-height: ${size === 'md' ? 277 : 265}px;
   overflow-y: auto;
-  max-width: 440px; // TODO we need a technique to identify menu position left or right
+  // TODO we need a technique to identify menu position left or right
```

Comment:
Maybe we could try extending the PositionInScreen hook? Or at least using it as e reference for this.